### DISCUSSION TITLE
Sensor Error

### DISCUSSION THREAD

**Discussion by Arakahn** [Rem0o/FanControl.Releases#2566] [Q&A]
Been having sensor errors since last Windows update :(

![image](https://github.com/Rem0o/FanControl.Releases/assets/43348379/bd051356-b77e-4e31-92b6-6afcc5e944db)

**Comment by Arakahn** [comment]
They appear without any problems in Core Temp. updated my Bios to the latest (Aorus Z790); when I boot it works for a little bit then it fails and CPU temp reads at 0 (while still reading fine in Core Temp)

**Comment by Rem0o** [comment]
From the looks of it it seems like it can't read anything.  Do you run the program as admin?

**Reply by Arakahn** [reply]
Yes; and if that was an Admin rights issue, I would assume that it would never read, but it reads at first but it stops after a while.

**Reply by Rem0o** [reply]
After a resume from sleep?

**Reply by Arakahn** [reply]
Not even. Sleep is disabled on my PC. It works after I reboot, then stops at some point, while I'm active on the PC" Just tried to restart it as Admin, just to be sure, same result

**Reply by Arakahn** [reply]
Just reinstalled Gigabyte Control Center (had uninstalled it when I switched to Fan Control), and it reads and controls the fans fine there as well,

**Reply by Arakahn** [reply]
Don't know if it can help (or if it is even relevant), running Windows 11 v.23H2, update KB5039212 was done yesterday, and the issue started not long after that

**Reply by mandala1974** [reply]
hi,i have the same problem from yesterday,never before.i have windows 10.Please,help me.Thanks

**Reply by Rem0o** [reply]
Any of you running kapersky AV by any chance?

**Reply by mandala1974** [reply]
i have kaspersky

**Reply by Rem0o** [reply]
https://github.com/Rem0o/FanControl.Releases/issues/2571

**Reply by mandala1974** [reply]
i close the kaspersky and problem continue

**Reply by mandala1974** [reply]
The problem is Kaspersky, I uninstalled it and now it works again. I have Kaspersky Plus version 21.17.7.539. Now I need to see how to configure Kaspersky. Has anyone already configured it and would be kind enough to help me? Thank you.
